plugin/PDFView.qml

```diff
@@ -132,6 +136,35 @@ SilicaFlickable {
         linkColor: Theme.highlightColor;
         pagePlaceholderColor: Theme.highlightColor;
 
+        onPageLayoutChanged: if (rememberPositionConfig.value) restorePageSettings()
```

Comment:
About PageRectangle(), well there are currently three information that are stored :
* page index, with a little adjustement because currentPage is the biggest page shown, but in this patch I wanted to restore the exact screen configuration, thus if we are showing page N but on top a bit of page N-1 is visible, I need to store N-1 in the database ;
* page position, *i.e.* the (left, top) coordinates, so in my previous example, I show the page N-1 but translate the view to a coordinate top= 0.85 for instance.
* zoom level, stored as the width of the PDFCanvas object in pixel.

I can drop the last information, but I think that having (index, top, left) is quite fine and easier for visual memory to associate state when leaving the document with the state when returning to the document. Additionally, having (index, top, left) fits well with the current state of the goToPage() function which waits for these info indeed.

And finally to return to your question, PageRectangle() is needed to obtain this tripplet.